src/monitoring/getStatus.js

```diff
@@ -0,0 +1,107 @@
+import fetch from "node-fetch"
+import { createRequire } from "module"; // Bring in the ability to create the 'require' method
+const require = createRequire(import.meta.url);
+globalThis.fetch = fetch
+
+
+let indexerQuery = `{ 
+        indexingStatuses  { 
+            subgraph synced health entityCount 
+            chains { 
+                latestBlock { number }
+                network
+            } 
+            fatalError {
+                message
+                block { hash number }
+                handler deterministic
+            }
+        } 
+    }`
+
+export class getDataToMonitorStatus {
+
+    static async getSubgraphData(indexerGraphQLURL) {
```

Comment:
avoid static methods. Make this instance method.
You can make object of the class and then call these methods